### PR TITLE
Bug 1168860 - [Settings] Initial usage of gaia-switch in the display panel

### PR DESCRIPTION
**Pull request by KevinGrandon**
https://bugzilla.mozilla.org/show_bug.cgi?id=1168860

### CODE REVIEW

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=e4aedd21ac4c93aca13a63d5654c1b7c81011565

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=abafde764738e73589ec2e5de90c5f5ce98d2a6b

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=20a76a6b2323b60b36d1f755ce80f9ee75c84dae

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=e4701b501936684d7f0df27c6e7d7d341df10d2e

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=6351059db10a93aedad0f0b544590b3ef960dc1f

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=7a170dd8f175304f88fc66955812810ad263109a

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=9cfb6ecb68aa71930d53564d94458106dbf82047

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=4d15dccf86ebad1429fc41f5c5f1b2e10ee8973a

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=58e5d0b71ee0bb194320368a8be7f3a87c79524b

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=159f286985c7ec6c75a6912eb9a414b3c2c7482f

File: apps/settings/style/lists.css [hunk 1]
```diff
@@ -18,6 +18,13 @@ ul li {
   color: #000;
 }
 
+/* Provide additional margin around gaia-switch to match spacing with legacy components */
+ul li > gaia-switch {
+  -moz-margin-start: 1.5rem;
+  /* The !important is necessary to override gaia-swtich scoped style. */
+  width: calc(100% - 1.5rem) !important;
+}
```
Comment (crh0716) [inline apps/settings/style/lists.css]: We should also override the text color with black.

File: apps/settings/index.html [hunk 2]
```diff
@@ -59,6 +59,8 @@
     <link rel="stylesheet" type="text/css" href="/shared/elements/gaia-icons/gaia-icons.css">
     <link rel="stylesheet" type="text/css" href="app://theme.gaiamobile.org/shared/elements/gaia-theme/gaia-theme.css">
     <script src="shared/elements/gaia-header/dist/gaia-header.js"></script>
+    <script defer src="/shared/js/component_utils.js"></script>
```
Comment (crh0716) [inline apps/settings/index.html]: nit: Please help remove this line[[1]] that loads the same file.

[1]: https://github.com/mozilla-b2g/gaia/blob/master/apps/settings/js/modules/panel_cache.js#L26

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=79f4c8afd78927f4eface4e4e74854a4a5581d65

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=7cd3549cfcc1d417ad671e3ea62014e3c08c900d

File: apps/settings/index.html (hunk 2 above)
Comment (KevinGrandon) [inline apps/settings/index.html]: Done, thanks.

File: apps/settings/style/lists.css (hunk 1 above)
Comment (KevinGrandon) [inline apps/settings/style/lists.css]: For now I've updated the component color, and in the future this should move to gaia-theme.

https://github.com/KevinGrandon/gaia/blob/bug_1168860_settings_display_panel_gaia_switch/shared/elements/gaia_switch/style.css#L22

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=ef7ea5408a6577185bd3afca73ca20954f32478c

**Pull request merged**